src/components/App/App.js

```diff
@@ -2,25 +2,59 @@ import React, { useState, useEffect } from 'react';
 import { getTips, addTip } from '../../ApiCalls';
 import TipJar from '../TipJar/TipJar';
 import Form from '../Form/Form';
+import Error from '../Error/Error';
+import Loader from '../Loader/Loader';
 import './App.css';
 
 const App = () => {
   const [advice, setAdvice] = useState([]);
+  const [error, setError] = useState('');
+  const [loading, setLoading] = useState('Loading tips...');
```

Comment:
is this the thing that Leta talked about avoiding having? Like can't we just check `advice.length` and then load from there? I know every example on the web has an isLoading boolean and I don't fully understand why its a problem....but I know Leta suggested we could achieve the same thing without another thing in state to keep track of

Reply:
Fixed this! you're absolutely right about this. It is unnecessary.